pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl

```diff
@@ -0,0 +1,116 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/TrimAdapters.wdl" as TrimAdapters
```

Comment:
Don't we typically use relative imports?

Reply:
This is temporary to let Allen's people. We need to change this before the PR merge

Reply:
Yes, they are updated now.

Reply:
I am still seeing the relative imports in this wdl, but you had said they were updated - did I misunderstand?